src/ovs/aca_vlan_manager.cpp

```diff
@@ -395,76 +395,105 @@ int ACA_Vlan_Manager::get_outports_unsafe(uint tunnel_id, string &outports)
   return overall_rc;
 }
 
-// query oam_port with tunnel_id
-uint ACA_Vlan_Manager::get_oam_server_port(uint tunnel_id)
+// create a neighbor port without specifying vpc_id and neighbor ID
+int ACA_Vlan_Manager::create_neighbor_outport(alcor::schema::NetworkType network_type,
```

Comment:
have we close on the approach to create remote IP specific outports on the source node? How do we manage the life time of it to delete it when it is done? Keep in mind a non-zeta port can also share this remote IP specific outports that's why it is using the neighbor IP to keep track of its life time.

Suggestion, can we simply use generic vxlan outport for both source and destination node with Zeta ports?